pyproject.toml

```diff
@@ -102,3 +102,9 @@ ignore = ["D200", "D212"]
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/flowcept"]
+
+[tool.hatch.build.targets.wheel.force-include]
+"resources/sample_settings.yaml" = "resources/sample_settings.yaml"
```

Comment:
If you are going to make the settings file required by the package then just put it in the source directory of the package. Also, I don't see any changes made to this sample_settings.yaml file. I think it had adapters defined for zambeze, mlflow, tensorboard, and dask which means all those dependencies would also be needed for the installation. I thought the goal was to provide a minimum settings file with no adapters?